### PR TITLE
support task-priorities when use RabbitMQ / Redis as broker

### PR DESCRIPTION
**Pull request by b1ngz**
According to the official docs, celery support [task-priorities](http://docs.celeryproject.org/en/latest/faq.html#does-celery-support-task-priorities) when use RabbitMQ or Redis as broker, and in [Advanced Options](http://docs.celeryproject.org/en/latest/userguide/calling.html#advanced-options) section, there is a priority option

> - priority
>
>   A number between 0 and 255, where 255 is the highest priority.
>
>   Supported by: RabbitMQ, Redis (priority reversed, 0 is highest).

So i add `priority` field for model `PeriodicTasks` to support task-priorities.

### CODE REVIEW

File: django_celery_beat/models.py [hunk 1]
```diff
@@ -281,6 +282,10 @@ class PeriodicTask(models.Model):
     routing_key = models.CharField(
         _('routing key'), max_length=200, blank=True, null=True, default=None,
     )
+    priority = models.PositiveIntegerField(
+        _('priority'), default=0, validators=[MaxValueValidator(255)],
```
[the remark this answers is not in the document]
Comment (b1ngz) [reply django_celery_beat/models.py]: You are right, `None` is better, i have changed it，see https://github.com/celery/django-celery-beat/pull/189/commits/923770afe818fd7f79da78b8975b9d56db9844d7

**Comment by b1ngz** [comment]
I saw [Travis CI build failed](https://travis-ci.org/celery/django-celery-beat/jobs/445142295) due to line too long in the new migration file

> /home/travis/build/celery/django-celery-beat/django_celery_beat/migrations/0006_periodictask_priority.py:10:80: E501 line too long (106 > 79 characters)

which code line is:

```
 ('django_celery_beat', '0005_add_solarschedule_events_choices_squashed_0009_merge_20181012_1416'),
```

The dependent squashed migration filename is too long. Any idea how to fix it?

**Comment by thedrow** [comment]
You should add a `# noqa` comment on that line.

**Comment by thedrow** [comment]
I just did it for you :). It should do the trick.

**Comment by b1ngz** [comment]
Thanks.  😫 But ci still fail due to 

> /home/travis/build/celery/django-celery-beat/django_celery_beat/tzcrontab.py:73:21: W504 line break after binary operator
/home/travis/build/celery/django-celery-beat/django_celery_beat/tzcrontab.py:74:21: W504 line break after binary operator
...

These code is already in master, should i make some change to fix it?

**Comment by thedrow** [comment]
If you can, please do.

**Comment by b1ngz** [comment]
[Line break before a binary operator is more readable](https://github.com/PyCQA/pycodestyle/issues/197#issuecomment-219742628), but after modified, the CI still failed.

After google for this, I found [the result of W503 and W504 are weird](https://github.com/spack/spack/pull/9627) which cause CI failed, so i add them to the ignore list.

**Comment by thedrow** [comment]
Thank you for your contribution.

**Pull request merged**